fs.c

```diff
@@ -1903,6 +1903,18 @@ static int FS_ChooseUserDir(userdirmode_t userdirmode, char *userdir, size_t use
 		if(homedir)
 		{
 			dpsnprintf(userdir, userdirsize, "%s/.%s/", homedir, gameuserdirname);
+#if defined(__unix__) || defined(__unix)
+			// Check for darkplaces config paths in this order: ~/.darkplaces, XDG_CONFIG_HOME/darkplaces, ~/.config/darkplaces.
+			if(access(userdir, F_OK) != 0)
+			{
+				// Check if XDG_CONFIG_HOME exists, test if absolute path.
+				const char *xdgconfdir = getenv("XDG_CONFIG_HOME");
```

Comment:
Note that `XDG_CONFIG_HOME` may be put on version control, and thus is not appropriate for in particular the `dlcache` subdirectory.

Any chance that can be fixed first?

Reply:
Failing that, I guess `~/.local/share/darkplaces` won't be _wrong_, but also not substantially better.